docs/Configuration.md

```diff
@@ -292,9 +292,11 @@ Default value: `0`.
 ### age_reuse_previous_test
 
 Positive integer.
-How old (in seconds) a previous test of the same zone name and parameters must
-be before we start a new test.
-Default value: `600`.
+
+The shelf life of a test in seconds after its creation.
+If a new test is requested for the same zone name and parameters within the
+shelf life of a previous test result, that test result is reused.
+Otherwise a new test request is enqueued.
```

Comment:
I think you removed the default value line by mistake.
(I like the rewording btw)

Reply:
Good spotting! I added it back.
I also also made a small edit to make age_reuse_previous_test to make its formatting more like the other simple properties.

> (I like the rewording btw)

I'm glad to hear that :)